cmd/root.go

```diff
@@ -61,4 +62,10 @@ func init() {
 	rootCmd.PersistentFlags().BoolP(argRootSkipVerify, "k", false, "skip SSL certificate verification")
 	rootCmd.PersistentFlags().StringP(argRootToken, "", "", "token file path")
 	rootCmd.PersistentFlags().BoolP(argRootVerbose, "v", false, "print verbose output")
+	rootCmd.AddCommand(loginCmd)
+	rootCmd.AddCommand(artifactsCmd)
+
+	if os.Getenv("MENDER_CLI_GEN_AUTOCOMPLETE") != "" {
```

Comment:
Any particular reason why this is not a command line argument instead?

Reply:
Poor decision making